doc.md

```diff
@@ -0,0 +1,110 @@
+# Hackseq 2017
+
+The system allows users to run graphical user interface application (e.g. `igv`), provides 100 GB hard memory and 20 GB soft memory, maps host users to Docker users inside containers and mounts team folders. 
+
+## Overview
+
+**Servers**: orca01.dmz.bcgsc.ca, orca02.dmz.bcgsc.ca
+* 5 teams per a server
+* a team shares a team’s folder
+* the Docker container per a team
+* each user has a home directory
+* ability to run GUI applications (X11)
+* the groups’ user name is the same as a name of the team
+
+## Group creation
+
+Add groups: `orca_users:*:1000:`, name of teams (e.g. `hackseq17_1:*:1002:`)
```

Comment:
Describe the purpose of these groups. Why do they exist, and what permissions does membership give to the user?

Reply:
Be specific about which group gives what access.

The `orca_users` group gives them read-only access to all the project folders.
The project group (e.g. `hackseq17_1`) gives them write access to that project folder. Their primary group determines the name of the container.